vignettes/a07_updating_rsyncrosim_scripts_to_v2.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Updating rsyncrosim scripts to version 2"
+
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Updating rsyncrosim scripts to version 2}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+params:
+  EVAL: !r identical(Sys.getenv("NOT_CRAN"), "true")
+---
+
+The introduction of <a href="https://syncrosim.com/studio/" target="_blank">SyncroSim Studio</a> brought some structural changes and new functionalities, which are reflected in the `rsyncrosim` package. Use the guide below to update your `rsyncrosim` version 1 scripts to be compatible with version 2.
+
+*Note*: <a href="https://syncrosim.com/studio-download/" target="_blank">SyncroSim v3.0.9</a> or higher is required to use `rsyncrosim v2.0.0` or higher. 
+
+## Updated functions
+
+### Installing/uninstalling packages
+
+Previously, the `addPackage` and `removePackage` functions were used to install and uninstall packages from SyncroSim. Now to install and uninstall packages from SyncroSim, use:
+
+```{r install packages, eval=FALSE}
+installPackage()   # Installs the selected package(s) to SyncroSim
+uninstallPackage() # Uninstalls the selected package(s) from SyncroSim
+```
+
+### Adding/removing packages
+
+Now you can use multiple SyncroSim packages in a single library. To add and remove packages from your library, use:
+
+```{r add package, eval=FALSE}
+addPackage()    # Adds package(s) to the library
+removePackage() # Removes package(s) from the library.
+```
+Note that when you remove a package from your library any associated datasheets will be removed as well.
+
+## Deprecated functions
+
+In SyncroSim 3, the concept of addon packages no longer exists. All addon packages have either been converted to standalone packages (e.g., `burnP3PlusPrometheus`) and can be added to a library without having to load a base package first, or incorporated directly into the base packages (e.g., `stsimsf` is now part of `stsim`). Therefore, the following functions have been removed:
+
+```{r deprecated, eval=FALSE}
+addon()
+disableAddon()
+enableAddon()
+```
+
+## Library
+
+`info(ssimLibrary)` no longer shows the *Package name*, *Current package version* and *Minimum package version*, and the Input and Output folder paths have been combined into the *Data folder*.
+
+The `package` argument in the `ssimLibrary(package)` function has been renamed to `packages`
+
+```{r library, eval=FALSE}
+myLibrary <- ssimLibrary(packages = c())
+```
+
+The `addon` and `template` arguments have also been removed.
```

Comment:
I would add this to above because it is unclear it relates to the `ssimLibrary()` function